salt_stack/salt/lambda-dev/files/makedomainenv

```diff
@@ -57,6 +57,13 @@ else
    cp makefile_LINUX makefile
    make
 
```

Comment:
We probably still need to update `makedomainenv` on the production lambda server.